File: Swift/Swift3基础/11-OC&Swift互调/SwiftCallOC/SwiftCallOC/CustomButton.m
//
//  CustomButton.m
//  SwiftCallOC
//
//  Created by 黄继平 on 2016/11/20.
//  Copyright © 2016年 PJStudio. All rights reserved.
//

#import "CustomButton.h"
#import "SwiftCallOC-Swift.h"

@implementation CustomButton

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
{
    ViewController *vc = [[ViewController alloc] init];
    vc.view.backgroundColor = [UIColor greenColor];
    
    [[UIApplication sharedApplication].keyWindow.rootViewController presentViewController:vc animated:YES completion:nil];
    
}

@end

File: Swift/Swift3基础/11-OC&Swift互调/1-OCCallSwift/OCCallSwift/CustomButton.swift
//
//  CustomButton.swift
//  OCCallSwift
//
//  Created by 黄继平 on 2016/11/20.
//  Copyright © 2016年 PJStudio. All rights reserved.
//

import UIKit

open class CustomButton: UIButton {

    open var customTitle : String = "" {
    
        didSet{
            setTitle(customTitle, for: .normal)
        }
    }
    
    open func test(){
        print("oc调用swift中的方法")
    }

}
